src/components/Profile/components/Identification/index.tsx

```diff
@@ -62,6 +66,7 @@ const Identification = ({ profile, myProf, isOnline, createSnackbar }: Props) =>
   const { updateImage } = useUserActions();
   const cropperRef = useRef<(ReactCropperElement & HTMLImageElement) | null>(null);
   const isStudent = profile.PersonType?.includes('stu');
+  const [showNameDialog, setShowNameDialog] = useState(false); // added line
```

Comment:
@abelyshev22 Please make the comment clearer.

Reply:
Also, please delete the extra line, 321, that's causing a warning (while you're in the file).

Reply:
Also line 428